style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+}
+
+.head__cont {
+  background-image: url(/img/vec-bg-1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: 100vh;
+}
+
+.nv__section {
+  /* border: 2px solid darkcyan; */
+  display: flex;
+  justify-content: space-between;
+  margin: 6% 10%;
+}
+
+#welcome {
+  padding-top: 3%;
+  font-size: 1.7rem;
+}
+
+#welcome > *,
+::before,
+::after {
+  text-decoration: none;
+  list-style: none;
+  color: #28352f;
+}
+
+.menu__button {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+}
+
+.menu__bar {
+  width: 2.5rem;
+  margin-top: 15%;
+  height: 5px;
+  position: relative;
+  background: #000;
+  transition: all 0.3s ease-in-out;
+  border-radius: 5px;
+  border: 0.1mm solid white;
+}
+
+.mn__title {
+  color: #172b4d;
+  margin: 0 auto;
+  width: 88%;
+  max-width: 500px;
+  text-align: center;
+  font-weight: bolder;
+  line-height: 3.2rem;
+
+  /* font-size: 10.5vw; */
+  font-size: 2.6rem;
+  margin-top: 11%;
+
+  /* border: 2px solid red; */
+}
```

Comment:
- Kindly stick to the Figma design with font and colors (background and font) defined for the template.😁👌
- you didn't add the whole background image please add it 😁👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/211907253-420b0e4a-908d-4fee-9ffe-ca664e09a339.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/211907243-85f6749a-4c08-4e31-9b17-5081e9530d72.png" width="200">|


😎️

🤝️